commands/visionCommand.py

```diff
@@ -0,0 +1,65 @@
+import wpilib
+import commands2
+import commands2.cmd
+import wpimath.controller
+
+from subsystems.photonVisionSubsystem import visionSub
+
+import constants
+
+import logging
+logger = logging.getLogger("Jhony")
+        
+class takeSnapShot(commands2.Command):
+    def __init__(self, visionSubsystem: visionSub) -> None:
+        super().__init__()
+        self.vision_sub = visionSubsystem
+
+    def initialize(self):
+        logger.info("Capturing Image")
+
+    def execute(self):
+        self.vision_sub.captureImage()
+
+    def isFinished(self):
+        # command does not finish it needs to be cancelled
+        return False
+
+    def end(self, interrupted: bool):
+        self.vision_sub.nothingCommand()
```

Comment:
`vision_sub.nothingCommand()` does nothing except returning `False`.  There doesn't seem to be any purpose in doing that at the end of this command or any other vision command.  I'd suggest removing it.  In this case, that would mean not having an `end()` method.